scylla/src/statement/prepared_statement.rs

```diff
@@ -247,6 +248,19 @@ impl PreparedStatement {
         self.config.timestamp
     }
 
+    /// Sets the client-side timeout for this statement.
+    /// If not None, the driver will stop waiting for the request
+    /// to finish after `timeout` passed.
+    /// Otherwise, default session client timeout will be applied.
+    pub fn set_request_timeout(&mut self, timeout: Option<Duration>) {
```

Comment:
Nit: `timeout: None` can be easily misinterpreted as "no timeout". I don't have an idea for a better alternative, though :(

Reply:
In discussion with @psarna, this behaviour was chosen as appropriate.

Reply:
yup - `None` here means "not specified timeout", hopefully nobody would get confused by that. An an unlimited timeout, which isn't exactly recommended, can be simulated by setting a multi-hour timeout if need be